e2e/isolated/isolated_test.go

```diff
@@ -0,0 +1,247 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package isolated_test
+
+import (
+	"errors"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	osExec "os/exec"
+
+	"os"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+)
+
+var _ = Describe("Isolated", func() {
+	Context("when creating a new app", func() {
+		var appInitErr error
+		BeforeAll(func() {
+			_, appInitErr = cli.AppInit(&client.AppInitRequest{
+				AppName: appName,
+				Tags: map[string]string{
+					"e2e-test": "isolated",
+				},
+			})
+		})
+
+		It("app init succeeds", func() {
+			Expect(appInitErr).NotTo(HaveOccurred())
+		})
+		It("app init creates a copilot directory", func() {
+			Expect("./copilot").Should(BeADirectory())
+		})
+		It("app ls includes new app", func() {
+			Eventually(cli.AppList, "30s", "5s").Should(ContainSubstring(appName))
+		})
+		It("app show includes app name", func() {
+			appShowOutput, err := cli.AppShow(appName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(appShowOutput.Name).To(Equal(appName))
+			Expect(appShowOutput.URI).To(BeEmpty())
+		})
+	})
+
+	Context("when deploying resources to be imported", func() {
```

Comment:
It seems like we delete the stack after app delete. Shall we create the vpc stack before doing app init? Because otherwise the stack we want to delete might not exist (we could ignore the error but not ideal 🤔 )

Here we just need to double check if the VPC stack is there before moving on